fic_modules/git.py

```diff
@@ -152,26 +156,30 @@ def filter_git_tag_bp(repository_name, repository_team, repository_path):
         .get("files-to-check")
     last_checked = last_check(repository_name)
     new_commit_dict = {"0": {"lastChecked": str(datetime.utcnow())}}
+
     if limit_checker():
-        new_commits = GIT.get_repo(repository_path).get_commits(since=last_checked)
+        new_commits = GIT \
+            .get_repo(repository_path) \
+            .get_commits(since=last_checked)
+
     for commit in new_commits:
         each_commit = {}
         switch = False
-        logger.info("Commit number: ", commit_number_tracker)
+        LOGGER.info("Commit number: " + str(commit_number_tracker))
         commit_number_tracker += 1
         files_changed_by_commit = [x.filename for x in commit.files]
-        logger.info(files_changed_by_commit)
-        logger.info(len(files_changed_by_commit))
+        LOGGER.info(files_changed_by_commit)
+        LOGGER.info(len(files_changed_by_commit))
         changed_file_number = 1
         for entry in files_changed_by_commit:
-            logger.info("changed file number: {} ".format(changed_file_number))
-            logger.info(entry)
+            LOGGER.info("changed file number: {} "
+                        .format(changed_file_number))
+            LOGGER.info(entry)
             changed_file_number += 1
             for scriptworkers in pathway:
-                logger.info("checking repo:{} ".format(scriptworkers))
-                number2 = 0
```

Comment:
Do we don't need this anymore?

Reply:
That `number2 = 0` was unused and the `logger.info("checking repo:{} ".format(scriptworkers))` logger line was just modified into `LOGGER.info("checking repo:{} ".format(scriptworkers))`